### ISSUE TITLE
Assign SNPs on chromosome 0

### ISSUE THREAD

**Issue by apriha** [apriha/snps#13]
Some newer [Family Tree DNA](https://www.familytreedna.com) famfinder files seem to have SNPs assigned to chromosome 0. Similar to `SNPs._assign_par_snps`, use the [RefSNP](https://api.ncbi.nlm.nih.gov/variation/v0/) API to assign each of these SNPs to a chromosome and update the SNP's position to be consistent with the assembly.

**Comment by willgdjones** [comment]
I have noticed that for AncestryDNA files, the `_assign_par_snps` step can take a substantial amount of time (> 1 minute), because of repeated API requests. I have currently bypassed this issue by setting `assign_par_snps=False`. Is there a way we can speed this up with the [VCF_endpoint](https://api.ncbi.nlm.nih.gov/variation/v0/#/VCF/post_vcf_file_set_rsids) like you suggest in #19.

**Comment by apriha** [comment]
Yeah, I agree that having a resource like discussed in #19 to lookup SNPs would help here as well and would reduce repeated API calls. I'll start looking into this as part of #42.
